lib/components/calendar/event_card.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class EventCard extends StatelessWidget {
+  EventCard({
+    Key? key,
+    required this.eventTitle,
+    required this.partner,
+    required this.time,
+    required this.room,
+  }) : super(key: key);
```

Comment:
Nadpisywanie key po : super to stare podejście, lepiej robić tak:

EventCard({
    super.key,
    required this.title,
    required this.partner,
    required this.time,
    required this.room,
  });